modules/meteroid/proto/api/invoices/v1/models.proto

```diff
@@ -23,7 +23,7 @@ message Invoice {
   string invoice_date = 4;
   string customer_id = 5;
   string customer_name = 6;
-  string subscription_id = 7;
+  optional string subscription_id = 7;
```

Comment:
@gaspb not sure if those changes break ui

Reply:
np i'm working on this view :)